### PR TITLE
simplify analysis_options.yaml usage

### PR DESCRIPTION
**Pull request by a14n**
This PR allows to avoid copy-pasting lint lists and simplify its handling and versioning.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,18 +30,33 @@ Here is how static analysis is used internally at Google:
...
    attempt to check in a further violation will be blocked.
-   
+
 ## Enabled Lints
 
 The currently enabled lints can be found in the sample
-[analysis_options.yaml](https://github.com/dart-lang/pedantic/blob/master/analysis_options.yaml).
+[analysis_options.yaml](https://github.com/dart-lang/pedantic/blob/master/lib/analysis_options.yaml).
+
+### Usage
```
Comment (davidmorgan) [inline README.md]: I think it looks a bit nicer without this header. Up to you.

Comment (a14n) [reply README.md]: Header removed.

**Comment by davidmorgan** [comment]
Thanks!

**Comment by a14n** [comment]
Could you please release a `1.1.0` with those changes?

**Comment by davidmorgan** [comment]
Of course. I'll aim to do that today.

**Comment by a14n** [comment]
Thanks :)

**Pull request merged**